keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+import argparse
+import pycparser
+
+def beautify_src_file(filename):
+    src = {
+        "before": [],
+        "keys": [],
+        "after": []
+    }
+
+    with open(filename) as f:
+        current_section = "before"
+        for line in f.read().split("\n"):
+            if current_section == "before":
+                if line == "const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {":
+                    current_section = "keys"
+                    src[current_section].append("const int keymaps[]={")
+                else:
+                    src[current_section].append(line)
+            elif current_section == "keys":
+                src[current_section].append(line)
+                if line == "};":
+                    current_section = "after"
+            else:
+                src[current_section].append(line)
+
+    output = "\n".join(src["before"]) + "\n" 
+    output += beautify_keys_section("\n".join(src["keys"])) + "\n"
+    output += "\n".join(src["after"]) + "\n"
+    return output
+
+def beautify_keys_section(src):
+    #parsed = pycparser.parse_file(c_filename)
+    parsed = pycparser.CParser().parse(src)
+    layer_output = []
+    for layer in parsed.children()[0][1].init.exprs:
```

Comment:
Break this up into variables so it's clear what `parsed.children()[0]` and `parsed.children()[0][1]` are. For example:
```suggestion
    keymap = parsed.children()[0]
    layers = keymap[1]
    for layer in layers.init.exprs:
```
(I don't know if it's actually the `keymap` or `layers`, I'm just guessing — hence why they should be variables.)